### PR TITLE
Added question

### PR DESCRIPTION
**Pull request by shubhsinha**
Checklist:

- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ ] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by shubhsinha** [comment]
added question

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -576,6 +576,16 @@ const sqlQuiz = [
...
+    Question:
+      "The best datatype to use to store a string with up to 255 characters is which of the following?",
+    Answer: "TINY TEST",
+    Distractor1: "TEXT",
+    Distractor2: "BLOB",
+    Distractor3: "BINARY",
+    Explanation:
+      "A string in TINY TEXT can be up to 255 characters or 255 bytes long.",
+  },
 ];
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:589]: This questions is missing a link field.
All questions are required to have a link.
Please use a freeCodeCamp resource, official documentation or wikipedia article

File: src/data/sql-quiz.ts [hunk 2]
```diff
@@ -576,6 +576,16 @@ const sqlQuiz = [
       "The ROUND() function takes in two parameters(number and decimals) and rounds the number to the number of decimals specified in the 2nd parameter.",
     Link: "https://dev.mysql.com/doc/refman/5.7/en/precision-math-rounding.html",
   },
+  {
+    Question:
+      "The best datatype to use to store a string with up to 255 characters is which of the following?",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:581]:
```suggestion
      "In SQL, which data type is used to store a string with up to 255 characters?",
```

File: src/data/sql-quiz.ts [hunk 3]
```diff
@@ -576,6 +576,16 @@ const sqlQuiz = [
...
+    Answer: "TINY TEST",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:582]: The last word should be text and it should be all one word

```suggestion
    Answer: "TINYTEXT",
```

File: src/data/sql-quiz.ts [hunk 4]
```diff
@@ -576,6 +576,16 @@ const sqlQuiz = [
...
+    Distractor1: "TEXT",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:583]:
```suggestion
    Distractor1: "STRING",
```

File: src/data/sql-quiz.ts [hunk 5]
```diff
@@ -576,6 +576,16 @@ const sqlQuiz = [
...
+    Distractor2: "BLOB",
+    Distractor3: "BINARY",
+    Explanation:
+      "A string in TINY TEXT can be up to 255 characters or 255 bytes long.",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:587]:
```suggestion
      "TINYTEXT can store up to 255 bytes and can be used for text like captions or summaries.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Pull request closed without merging**